### PR TITLE
fix(preset-mini): restore inset position, add inset tests

### PR DESCRIPTION
**Pull request by sondh0127**
fix https://github.com/antfu/unocss/issues/394

### CODE REVIEW

**Comment by chu121su12** [comment]
I'm sorry the latest PR for the Issue. Could you please make the rule preserve the order of `inset-[unit]` (without x/y)?

**Comment by sondh0127** [comment]
> I'm sorry the latest PR for the Issue. Could you please make the rule preserve the order of `inset-[unit]` (without x/y)?

No problem. Your work was fantastic.
I fixed the order. I add a map for zero: 0rem => 0. Is that look good?

**Comment by chu121su12** [comment]
Maybe split the 0 handling to another PR since most sizing rules with handler may benefit from this.

File: packages/preset-mini/src/rules/position.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ function handleInsetValues([, d, v]: string[]): CSSEntries | undefined {
 }
 
 export const insets: Rule[] = [
-  [/^(?:position-|pos-)?inset-()(.+)$/, handleInsetValues],
+  [/^(?:position-|pos-)?inset-()(.+)$/, ([, , v]) => ({ inset: handleInsetValue(v) })],
```
Comment (chu121su12) [inline packages/preset-mini/src/rules/position.ts:99]: The empty capture `()` can be removed.

Comment (sondh0127) [reply packages/preset-mini/src/rules/position.ts]: Alright, I'm new to this. Thanks for figuring it out

**Comment by chu121su12** [comment]
Thanks!

**Pull request merged**